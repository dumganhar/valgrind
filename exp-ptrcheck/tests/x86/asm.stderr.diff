1,3c1
< 
< 
< ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
---
> valgrind: ./asm: No such file or directory
