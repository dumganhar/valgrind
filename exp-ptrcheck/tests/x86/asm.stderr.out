valgrind: ./asm: No such file or directory
